src/main.js

```diff
@@ -19,22 +25,16 @@ function getRandomIndex(array) {
 }
 
 function displayCover() {
-  getRandomCovers();
-  // see a cover with a randomly selected image and taglin with 2 random discriptors
-  // loop through array of cover images using getRandomIndex(covers) function
-  // repeat that for title and 2 discriptors
-}
-
-function getRandomCovers() {
-  getRandomIndex(covers);
+  coverImage.src = currentCover.cover;
```

Comment:
We need to look into why its not changing.

Reply:
Thanks Amanda! Glad we were able to take a look at it and figure out what is happening.